draft-ietf-acme-acme.md

```diff
@@ -306,36 +306,25 @@ JWS objects sent in ACME requests MUST meet the following additional criteria:
 * The JWS MUST NOT have the value "none" in its "alg" field
 * The JWS Protected Header MUST include the following fields:
   * "alg"
-  * "jwk"
+  * "kid" (except for new-reg and revoke-cert requests).
   * "nonce" (defined below)
   * "url" (defined below)
 
-Note that this implies that GET requests are not authenticated.  Servers MUST
+The "kid" field should contain a registration URL received by POSTing to the
+new-reg resource. The "kid" field is not required when POSTing to the new-reg
+resource, because no such URL is available to the client yet. The "kid" field
+may be omitted for revoke-cert requests if they are signed by the
+certificate key rather than an account key.
+
+Note that authentication via signed POST implies that GET requests are not
+authenticated.  Servers MUST
 NOT respond to GET requests for resources that might be considered sensitive.
 
 In the examples below, JWS objects are shown in the JSON or flattened JSON
 serialization, with the protected header and payload expressed as
 base64url(content) instead of the actual base64-encoded value, so that the content
 is readable.  Some fields are omitted for brevity, marked with "...".
 
-## Equivalence of JWKs
```

Comment:
Good point. I've left this section in for now, we can take another look at key roll-over separately.